projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -38,7 +37,7 @@
   void disablePlaceholder();
   void setItemFactory(Supplier<SourceT> itemFactory);
   void setSeparator(Character ch);
-  void replaceWithNewOnRemove(Predicate<SourceT> shouldReplace);
+  void setReplaceNonemptyWithNewOnRemove(boolean replaceWithNew);
```

Comment:
@timzam Multiple tests fail. Besides simple tests from `projectional`, there are failing tests in `census`. We can modify them to pass, but this may be not safe. What do you think?